core/src/main/java/org/pentaho/di/connections/utils/VFSConnectionTestOptions.java

```diff
@@ -0,0 +1,57 @@
+
+/*! ******************************************************************************
+ *
+ * Pentaho Data Integration
+ *
+ * Copyright (C) 2024 by Hitachi Vantara : http://www.pentaho.com
+ *
+ *******************************************************************************
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.pentaho.di.connections.utils;
+
+/**
+ * This class contains the options to ignore root path for a VFS Connection.
+ */
```

Comment:
Statement should not focus on the current single option, but instead on the general purpose of the class.
For example:
```java
/**
 * This class contains options that control the testing of VFS connections.
 */
```

Reply:
addressed.